src/momento/simple_cache_client_async.py

```diff
@@ -249,6 +268,153 @@ async def delete(self, cache_name: str, key: TScalarKey) -> CacheDeleteResponse:
+        ttl: CollectionTtl = CollectionTtl.from_cache_ttl(),
+    ) -> CacheDictionaryIncrementResponse:
+        """Add an integer quantity to a dictionary value.
+
+        Args:
+            cache_name (TCacheName): Name of the cache to store the dictionary in.
+            dictionary_name (TDictionaryName): Name of the dictionary to increment in.
+            field (TDictionaryField): The field to increment.
+            amount (int, optional): The quantity to add to the value. May be positive, negative, or zero. Defaults to 1.
+            ttl (CollectionTtl, optional): TTL for the dictionary in cache. This TTL takes precedence over the TTL
+                used when initializing a cache client. Defaults to client TTL.
+                Defaults to CollectionTtl.from_cache_ttl().
+
+        Returns:
+            CacheDictionaryIncrementResponse: result of the increment operation.
+        """
+        pass
+
+    async def dictionary_set_field(
+        self,
+        cache_name: TCacheName,
+        dictionary_name: TDictionaryName,
+        field: TDictionaryField,
+        value: TDictionaryValue,
+        ttl: CollectionTtl = CollectionTtl.from_cache_ttl(),
+    ) -> CacheDictionarySetFieldResponse:
+        """Set the dictionary field to a value with a given time to live (TTL) seconds.
+
+        Args:
+            cache_name (TCacheName): Name of the cache to store the dictionary in.
+            dictionary_name (TDictionaryName): Name of the dictionary to set.
+            field (TDictionaryField): The field in the dictionary to set.
+            value (TDictionaryValue): The value to be stored.
+            ttl (CollectionTtl, optional): TTL for the dictionary in cache.
+                This TTL takes precedence over the TTL used when initializing a cache client.
+                Defaults to CollectionTtl.from_cache_ttl().
+
+        Returns:
+            CacheDictionarySetFieldResponse: result of the set operation.
+        """
+        pass
+
+    async def dictionary_set_fields(
+        self,
+        cache_name: TCacheName,
+        dictionary_name: TDictionaryName,
+        items: TDictionaryItems,
+        ttl: CollectionTtl = CollectionTtl.from_cache_ttl(),
+    ) -> CacheDictionarySetFieldsResponse:
+        """Set several dictionary field-value pairs in the cache.
+
+        Args:
+            cache_name (TCacheName): Name of the cache to perform the lookup in.
+            dictionary_name (TDictionaryName): Name of the dictionary to set.
+            items (TDictionaryItems): Field value pairs to store.
+            ttl (CollectionTtl, optional): TTL for the dictionary in cache.
+                This TTL takes precedence over the TTL used when initializing a cache client.
+                Defaults to CollectionTtl.from_cache_ttl().
+
+        Returns:
+            CacheDictionarySetFieldsResponse: result of the set fields operation.
+        """
+        pass
+
+    async def dictionary_remove_field(
```

Comment:
I didn't notice `dictionary_set_fields` and `dictionary_set_field` because this is out of order.

Reply:
Changed in [27d8d70](https://github.com/momentohq/client-sdk-python/pull/229/commits/27d8d7038ec90ab87902dece3d14bd0a9fe0ec10)